.ci/pytorch/test.sh

```diff
@@ -1126,10 +1150,11 @@ elif [[ "${BUILD_ENVIRONMENT}" == *rocm* && -n "$TESTS_TO_INCLUDE" ]]; then
   install_torchvision
   test_python
   test_aten
-elif [[ "${BUILD_ENVIRONMENT}" == *xpu* && -n "$TESTS_TO_INCLUDE" ]]; then
-  install_torchvision
-  test_python
-  test_aten
+elif [[ "${BUILD_ENVIRONMENT}" == *xpu* ]]; then
+  #install_torchvision
+  #test_python
+  #test_aten
```

Comment:
This is an open, and for initial quick test purpose we just keep `test_xpu_bin` in here

Reply:
I mean to add TODO here to express it as an open. Otherwise, other contributors do not know what's the exact meaning of these comments. You can refine it by `#TODO: we will install torchvision(install_torchvision) and run python and aten test cases(test_python & test_aten) gradually`